src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,69 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+//Lotto를 관리하는 일급 컬렉션
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    //automated constructor
+    public Lotto() {
+        List<LottoNumber> cardinateNumbers = new ArrayList<>();
+        for(int i=1; i<=45; i++){
+            cardinateNumbers.add(new LottoNumber(i));
+        }
+        Collections.shuffle(cardinateNumbers);
+        this.numbers = cardinateNumbers.subList(0,6);
+        this.numbers.sort(LottoNumber::compareTo);
+    }
+
+    //manual constructor
+    public Lotto(List<Integer> numbers){
+        List<LottoNumber> lottoNumbers = new ArrayList<>();
+        for (Integer number : numbers) {
+            lottoNumbers.add(new LottoNumber(number));
+        }
+        validateLottoNumbers(lottoNumbers);
+        this.numbers = lottoNumbers; // 검증 후 할당
```

Comment:
검증을 제일 먼저 해주면 유효성이 맞지 않을 때 불필요한 로직을 실행하지 않아도 되겠죠?
```suggestion
        validateLottoNumbers(numbers);
        List<LottoNumber> lottoNumbers = new ArrayList<>();
        for (Integer number : numbers) {
            lottoNumbers.add(new LottoNumber(number));
        }
        this.numbers = lottoNumbers; // 검증 후 할당
```